ui/src/components/search/Search.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { css } from '@emotion/react';
 import Component from 'react-component-component';
-import { Aggregations } from '@overture-stack/arranger-components';
-// import {
-//Aggregations,
-// CurrentSQON,
-// Table,
-// } from '@arranger/components/dist/Arranger';
 import SplitPane from 'react-split-pane';
+import {
+  Aggregations as ArrangerAggregations,
```

Comment:
curious about the purpose/necessity to rename this component

Reply:
Eh left over from early setup, (distinguish btwn old v new) can be removed